homeassistant/components/renault/button.py

```diff
@@ -71,4 +71,11 @@ async def async_press(self) -> None:
         name="Start charge",
         requires_electricity=True,
     ),
+    RenaultButtonEntityDescription(
+        async_press=lambda x: x.vehicle.set_charge_stop(),
```

Comment:
Have you tested this feature?

Feedback on the renault-api library has been inconsistent, and the function doesn't appear to work on all vehicles (does it actually work on yours?)

Reply:
The service worked on my Dacia Spring. I will test the button today.

Reply:
Just tested it and worked with my car.

Dacia spring.

Reply:
Great can you please adjust tests accordingly?

Reply:
It is the same library so I think it's better to have a single integration.
Even some Renault vehicles don't support schedule charge...
It is better to adjust documentation for this